wikiclue/src/styles/dailyPageStyles.css

```diff
@@ -0,0 +1,82 @@
+.daily-page {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	margin-top: 5rem;
+}
+
+h1 {
+	font-size: 2rem;
+}
+
+.daily-title {
+    text-align: center;
+}
+
+.search-container {
```

Comment:
Any chance we can modify the width here so it isn't 100% the width of the screen on mobile?

Reply:
Fixed